src/coreclr/jit/emit.h

```diff
@@ -3146,10 +3146,10 @@ class emitter
     bool emitFullGCinfo;  // full GC pointer maps?
     bool emitFullyInt;    // fully interruptible code?
 
-    regMaskTP emitGetGCRegsSavedOrModified(CORINFO_METHOD_HANDLE methHnd);
+    AllRegsMask emitGetGCRegsSavedOrModified(CORINFO_METHOD_HANDLE methHnd);
 
     // Gets a register mask that represent the kill set for a NoGC helper call.
-    regMaskTP emitGetGCRegsKilledByNoGCCall(CorInfoHelpFunc helper);
+    AllRegsMask emitGetGCRegsKilledByNoGCCall(CorInfoHelpFunc helper);
```

Comment:
Seems like this might be able to be
```suggestion
    const AllRegsMask& emitGetGCRegsKilledByNoGCCall(CorInfoHelpFunc helper);
```